freqtrade/strategy/interface.py

```diff
@@ -531,12 +558,26 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
                        and self.min_roi_reached(trade=trade, current_profit=current_profit,
                                                 current_time=date))
 
+        sell_signal = SellType.NONE
+        custom_reason = None
         if (ask_strategy.get('sell_profit_only', False)
                 and current_profit <= ask_strategy.get('sell_profit_offset', 0)):
             # sell_profit_only and profit doesn't reach the offset - ignore sell signal
-            sell_signal = False
-        else:
-            sell_signal = sell and not buy and ask_strategy.get('use_sell_signal', True)
+            pass
+        elif ask_strategy.get('use_sell_signal', True) and not buy:
+            if sell:
+                sell_signal = SellType.SELL_SIGNAL
+            else:
+                custom_reason = strategy_safe_wrapper(self.custom_sell, default_retval=False)(
+                    trade.pair, trade, date, current_rate, current_profit)
+                if custom_reason:
+                    sell_signal = SellType.CUSTOM_SELL
+                    if isinstance(custom_reason, bool):
+                        custom_reason = None
+                    elif isinstance(custom_reason, str):
+                        if len(custom_reason) > CUSTOM_SELL_MAX_LENGTH:
+                            raise OperationalException('Custom sell reason returned '
+                                                       'from custom_sell is too long.')
```

Comment:
i'm not sure about raising an exception here - assuming a nested `custom_sell()` method which only hits a branch every once in a while ... 

So we'd risk a bot running for weeks - and then crash with `OperationalException` because of this ... 
maybe we can trim it to the allowed characters and warn about it instead?

that way it'll be "noisy as hell" in backtesting  should you hit that (intentional) - but won't break a live-bot because of a too long given reason.